src/platform-implementation-js/dom-driver/gmail/views/gmail-thread-row-view.ts

```diff
@@ -317,27 +318,46 @@ class GmailThreadRowView {
     this._userView = userView;
   }
 
-  getCounts(): Counts {
-    let counts = this._counts;
-
-    if (!counts) {
-      const recipientsElement = querySelector(this._elements[0], 'td div.yW');
-
-      const draftCount = recipientsElement.querySelectorAll('.boq').length;
-      const messageCountMatch = recipientsElement.querySelector('.bx0');
-      const messageCount =
-        messageCountMatch && messageCountMatch.innerHTML
-          ? +messageCountMatch.innerHTML
-          : draftCount
-          ? 0
-          : 1;
-      counts = this._counts = {
-        messageCount,
-        draftCount,
-      };
+  #getCounts(): Counts {
+    if (this.#counts) {
+      return this.#counts;
+    }
+
+    /**
+     * If the SDK is used alongside the https://chrome.google.com/webstore/detail/mailtrack-for-gmail-inbox/ndnaehgpjlnokgebbaldlmgkapkpjkkb/utm_source/gmail/utm_medium/signature/utm_campaign/signaturevirality/th138p9fIBkLHUhniapN extension, the recipientsElement will sometimes be null.
+     */
+    let recipientsElement;
+
+    try {
+      recipientsElement = querySelector(this._elements[0], 'td div.yW');
+    } catch (e: unknown) {
+      if (e instanceof SelectorError) {
+        this._driver.logger.errorSite(e);
+      } else {
+        throw e;
+      }
     }
```

Comment:
👍

Reply:
Seems safe, and looks good to me. I'd would like to see @Macil chime in though since he knows the ins and outs better.